src/Controllers/Content.php

```diff
@@ -86,6 +86,24 @@ public function index()
             $response->send();
         }elseif ( isset($_SESSION['lti']['issuer_client']) ) {
             $custom_email_id = ParamValidate::getKeyInCustomFields($_SESSION, 'person_email_primary');
+            if (isset($_SESSION['lti_post']['placement']) && $_SESSION['lti_post']['placement'] == 'canvas_sso') {
```

Comment:
Use "===" operator if possible.

Reply:
Also do we need to check for "$_SESSION['lti_post']", if it is set as well.

Reply:
> Also do we need to check for "$_SESSION['lti_post']", if it is set as well.

isset($_SESSION['lti_post']['placement']) will do the same, doesn't it?